### ISSUE TITLE
Hope there will be 3D version

### ISSUE THREAD

**Issue by WonderSimiliar** [HuXiaoling/imageSeg-2.5D_topo#5]
Hi, Xiaoling
Thanks for your excellent work!
Hope there will be 3D tensorflow version of it.

**Comment by HuXiaoling** [comment]
Hi @WonderSimiliar, thanks for your interest in our work!

We are preparing the true 3D version based on PyTorch, but also there is someone trying to incorporate the topological loss into Tensorflow framework. The related repositories are under construction and stay tuned:)

**Comment by WonderSimiliar** [comment]
> Hi @WonderSimiliar, thanks for your interest in our work!
> 
> We are preparing the true 3D version based on PyTorch, but also there is someone trying to incorporate the topological loss into Tensorflow framework. The related repositories are under construction and stay tuned:)

Thanks for your time！
